python_scripts/cross_validation_grouping.py

```diff
@@ -18,8 +17,8 @@
 data, target = digits.data, digits.target
 
 # %% [markdown]
-# We will recreate the same model used in the previous notebook: a logistic
-# regression classifier with a preprocessor to scale the data.
+# We create a model consisting of a logistic regression classifier with a
+# preprocessor to scale the data.
```

Comment:
We might want to explain why not using `StandardScaler`: some pixel have really low variance leading to undesired scale. Instead, we use use case knowledge where a pixel is encoded between 0 and max values and thus `MinMaxScaler` is a better scaling in this case.